src/main/features/core/lyrics/lrclib.ts

```diff
@@ -10,7 +10,7 @@ import {
 import { orderSearchResults } from './shared';
 
 const FETCH_URL = 'https://lrclib.net/api/get';
-const SEEARCH_URL = 'https://lrclib.net/api/search';
+const SEARCH_URL = 'https://lrclib.net/api/search';
```

Comment:
This is a separate issue, and should probably be done in a separate PR (or I can just fix this)